lawnchair/src/app/lawnchair/nexuslauncher/OverlayCallbackImpl.kt

```diff
@@ -0,0 +1,136 @@
+package app.lawnchair.nexuslauncher
+
+import android.app.Activity
+import android.content.Context
+import android.content.pm.ApplicationInfo
+import android.os.Bundle
+import app.lawnchair.FeedBridge
+import app.lawnchair.LawnchairLauncher
+import app.lawnchair.preferences2.PreferenceManager2
+import com.android.launcher3.Utilities
+import com.android.systemui.plugins.shared.LauncherOverlayManager
+import com.android.systemui.plugins.shared.LauncherOverlayManager.LauncherOverlay
+import com.android.systemui.plugins.shared.LauncherOverlayManager.LauncherOverlayCallbacks
+import com.google.android.libraries.launcherclient.ISerializableScrollCallback
+import com.google.android.libraries.launcherclient.LauncherClient
+import com.google.android.libraries.launcherclient.LauncherClientCallbacks
+import com.google.android.libraries.launcherclient.LauncherClientService
+import com.google.android.libraries.launcherclient.StaticInteger
+import com.patrykmichalik.opto.core.firstBlocking
+
+/**
+ * Implements [LauncherOverlay] and passes all the corresponding events to [LauncherClient],
+ * see [LauncherClientService.setClient].
+ *
+ * Implements [LauncherClientCallbacks] and sends all the corresponding callbacks to [ ].
+ */
```

Comment:
The document is missing the reference to `Launcher` like the original java file.

```suggestion
import com.android.launcher3.Launcher
import com.android.launcher3.Utilities
import com.android.systemui.plugins.shared.LauncherOverlayManager
import com.android.systemui.plugins.shared.LauncherOverlayManager.LauncherOverlay
import com.android.systemui.plugins.shared.LauncherOverlayManager.LauncherOverlayCallbacks
import com.google.android.libraries.launcherclient.ISerializableScrollCallback
import com.google.android.libraries.launcherclient.LauncherClient
import com.google.android.libraries.launcherclient.LauncherClientCallbacks
import com.google.android.libraries.launcherclient.LauncherClientService
import com.google.android.libraries.launcherclient.StaticInteger
import com.patrykmichalik.opto.core.firstBlocking

/**
 * Implements [LauncherOverlay] and passes all the corresponding events to [LauncherClient],
 * see [LauncherClientService.setClient].
 *
 * Implements [LauncherClientCallbacks] and sends all the corresponding callbacks to [Launcher].
 */
```